.eslintrc

```diff
@@ -67,6 +67,9 @@
     "react/no-deprecated": 2,
     "react/no-direct-mutation-state": 2,
 
+    /* TODO | DISABLED TEMPORARILY */
+    "react/sort-comp": 0,
```

Comment:
Why this is fine, [from the docs](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/sort-comp.md#when-not-to-use-it):

> This rule is a formatting preference and not following it won't negatively affect the quality of your code. If components organisation isn't a part of your coding standards, then you can leave this rule off.

Reply:
This rule will become active in the airbnb plugin once we upgrade all related dependencies. It's been disabled temporarily in the meantime.